book/coloring_and_theming.md

```diff
@@ -217,9 +213,33 @@ Example: `{ fg: "#ff0000" bg: "#0000ff" attr: b }`
 - background of blue in "#hex" format
 - attribute of bold abbreviated
 
-## Primitive Values
+### Closure
+
+Note: Closures are only executed for table output. They do not work in other contexts like for `shape_` configurations, when printing a value directly, or as a value in a list.
 
----
+For example:
+
+```nu
+$env.config.color_config.filesize = {|x| if $x == 0b { 'dark_gray' } else if $x < 1mb { 'cyan' } else { 'blue' } }
+$env.config.color_config.bool = {|x| if $x { 'green' } else { 'light_red' } }
+{a:true,b:false,c:0mb,d:0.5mb,e:10mib}
+```
+
+prints
+
+```nu
+╭───┬───────────╮
+│ a │ true      │
+│ b │ false     │
+│ c │ 0 B       │
+│ d │ 488.3 KiB │
+│ e │ 10.0 MiB  │
+╰───┴───────────╯
+```
+
+with a green `true`, a red `false`, a grey `0 B`, a cyan `488.3 KiB`, and a blue `10.0 MiB`.
```

Comment:
The color names don't match the definition above
```suggestion
with a green `true`, a light_red `false`, a dark_grey `0 B`, a cyan `488.3 KiB`, and a blue `10.0 MiB`.
```

Reply:
I deliberately did not use the symbol names. As a description for humans, I chose the simplest description of color names. Because there are no different heues displayed in this output. Just one kind of red and one kind of grey.

If you want it specific, I would at least use normal speak and not the symbols

```suggestion
with a green `true`, a light red `false`, a dark grey `0 B`, a cyan `488.3 KiB`, and a blue `10.0 MiB`.
```

Reply:
I just thought it was confusing because you're saying it's a different color than you specified above. I could live with your suggestion because at least someone can compare and agree that the setting worked based on the names alone.